api/src/services/fields.ts

```diff
@@ -37,10 +37,20 @@ import { getCollectionRelationList } from './fields/get-collection-relation-list
 import { ItemsService } from './items.js';
 import { PayloadService } from './payload.js';
 import { RelationsService } from './relations.js';
+import { isSystemField } from '@directus/system-data';
+import Joi from 'joi';
 
 const systemFieldRows = getSystemFieldRowsWithAuthProviders();
 const env = useEnv();
 
+export const systemFieldUpdateSchema = Joi.object({
```

Comment:
Could we use zod here instead?

Reply:
We can but then i'd rather also updatd the other Joi uses in this file, I just copied an existing validation and made some alterations felt overkill to make 1 zod exception.